source/_integrations/text.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: Text
+description: Instructions on how to set up text entities within Home Assistant.
+ha_category:
+  - Text
+ha_release: '2022.12'
+ha_domain: text
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+  - '@raman325'
+ha_integration_type: entity
+---
+
+The Text integration is built for the controlling and monitoring of text values on devices.
+
+Text entities are created automatically by integrations that support them.
```

Comment:
This makes it a little more clear that you can't just spawn these manually, while also providing an alternative that can be used manually.

```suggestion
Text entities cannot be implemented manually, but can be provided by other integrations. If you are looking for a way to create a text entity, please take a look at the [Text helper](/integrations/input_text).
```